### PR TITLE
chore: Adding local testing documentation to contributing guide.

### PR DESCRIPTION
**Pull request by ddixit14**
Hi team, just adding some stuff to the guide that maybe useful for contributors.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,5 +54,23 @@ accept your pull requests.
...
-[node]: https://nodejs.org/en/
\ No newline at end of file
+        
+## Testing a new feature using CLI
+
+1. After you've written some new code, in order to test it out, you can use the [CLI][CLI]. 
+   
+   For example, the CLI command:
+   
+   `release-please release-pr
```
Comment (chingor13) [inline CONTRIBUTING.md:64]:
```suggestion
   `release-please release-pr \
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,5 +54,23 @@ accept your pull requests.
...
+   
+   For example, the CLI command:
+   
+   `release-please release-pr 
+   --token=$GITHUB_TOKEN \
+   --repo-url=<owner>/<repo> [extra options]`
+   
+   can be run as from the root of the source code as:
+ 
+   `node build/src/bin/release-please.js release-pr
```
Comment (chingor13) [inline CONTRIBUTING.md:70]:
```suggestion
   `node build/src/bin/release-please.js release-pr \
```

**Review by chingor13 — CHANGES_REQUESTED** [review]
Thanks, LGTM, with the minor change to add `\` for the multi-line bash command

File: CONTRIBUTING.md (hunk 1 above)
Comment (suztomo) [reply CONTRIBUTING.md:64]: @ddixit14  The inline code syntax is strange. It has unnecessary backslash.

image

Use multiline code syntax with backslash.

Comment (suztomo) [inline CONTRIBUTING.md:64]: @ddixit14 Is there any step to create the "release-please" command?

Comment (ddixit14) [reply CONTRIBUTING.md:64]: No, no step to create the command.

**Comment by ddixit14** [comment]
Apologies, this got closed on its own because I deleted the fork. Reopening it for @chingor13's approval.

**Comment by suztomo** [comment]
Can you check the formatted view? I don't think you fixed the problem.

File: CONTRIBUTING.md (hunk 1 above)
Comment (suztomo) [reply CONTRIBUTING.md]: Then how to create the `release-please` command from the source?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -54,5 +54,23 @@ accept your pull requests.
 1.  Lint (and maybe fix) any changes:
 
         npm run fix
-
-[node]: https://nodejs.org/en/
\ No newline at end of file
+        
+## Testing a new feature using CLI
+
+1. After you've written some new code, in order to test it out, you can use the [CLI][CLI].
```
Comment (suztomo) [inline CONTRIBUTING.md:60]: Can you add the step to generate `release-please` command?

**Comment by ddixit14** [comment]
Closing this in favor of #1743.

**Pull request closed without merging**